src/App.jsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
 import './App.css';
 import Headers from './components/Headers';
 import Footers from './components/Footers';
 import PlaceBet from './components/PlaceBet';
 
-function App() {
-  return (
+
+const Hi = () => (
+  <>
+    Hello there sexy :)
+  </>
+);
+
+const App = () => (
+  <Router>
     <div className="App">
       <Headers />
       <div className="container">
-        <PlaceBet />
+        <Route path="/" exact component={PlaceBet} />
+        <Route path="/toss" component={Hi} />
```

Comment:
Routes not starting with `#` won't probably work on github pages, but we can fix that later